AGGREGATE.md

```diff
@@ -91,7 +91,12 @@ Registering sources eligible for aggregate reporting entails adding a new
     "geoValue": "0x5" // Source-side geo region = 5 (US), out of a possible ~100 regions
   },
 
-  "aggregatable_report_window": "86400"
+  "aggregatable_report_window": "86400",
```

Comment:
Shall we move this to a separate section for this new feature, similar to https://github.com/WICG/attribution-reporting-api/blob/main/AGGREGATE.md#optional-flexible-contribution-filtering-with-filtering-ids?